### PR TITLE
Client web [#7035] Simple homepage Space blocks

### PR DESCRIPTION
**Pull request by reactoholic**
Simple homepage Space blocks

## Release Notes

- **New Features**
  - Introduced a new dashboard component for displaying hierarchical space memberships.
  - Added a button in the Recent Journeys List for navigating to see more options.

- **Improvements**
  - Enhanced translation files for better user engagement and clarity.
  - Updated the MyMembershipsDialog for improved flexibility in displaying data and icons.

- **Bug Fixes**
  - Resolved inconsistencies in the GraphQL queries and improved formatting for better readability.

- **Chores**
  - Removed outdated environment variable files and updated the `.gitignore` for better file management.

### CODE REVIEW

**Comment by reactoholic** [comment]
@coderabbitai review

File: .build/docker/.env.base [hunk 1]
```diff
@@ -1,11 +0,0 @@
-VITE_APP_VERSION=0.74.0
```
Comment (bobbykolev) [inline .build/docker/.env.base:1]: it looks like you're deleting the file. this is not what we want

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@alkemio/client-web",
-  "version": "0.76.0",
+  "version": "0.76.1",
```
Comment (bobbykolev) [inline package.json:3]: please revert to the "0.76.0" like in the target branch (both package and package-lock)

File: src/main/topLevelPages/myDashboard/myMemberships/MyMembershipsDialog.tsx [hunk 3]
```diff
@@ -1,80 +1,69 @@
...
-    myMembershipStatus?: CommunityMembershipStatus;
-  };
-  childMemberships?: MembershipProps[];
-}
 
-const MyMembershipsDialog = ({ open, onClose }: MyJourneysDialogProps) => {
-  const { t } = useTranslation();
+  showFooterText?: boolean;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  Icon?: ReactElement<SvgIconProps, string | JSXElementConstructor<any>>; // This any type must stay here.
```
Comment (bobbykolev) [inline src/main/topLevelPages/myDashboard/myMemberships/MyMembershipsDialog.tsx:28]: we're passing icons in many components, let's be strict with something like  ComponentType<SvgIconProps>

File: .build/docker/.env.base (hunk 1 above)
Comment (reactoholic) [reply .build/docker/.env.base:1]: Yes, I need to fix this ebcauses locally `git` isn't telling me it's deleting the file.

File: src/main/topLevelPages/myDashboard/myMemberships/MyMembershipsDialog.tsx [hunk 4]
```diff
@@ -1,80 +1,68 @@
...
+import { ExpandableSpaceTree } from './ExpandableSpaceTree';
+import RouterLink from '../../../../core/ui/link/RouterLink';
+
 import { gutters } from '../../../../core/ui/grid/utils';
 import { useCreateSpaceLink } from '../useCreateSpaceLink/useCreateSpaceLink';
 
-interface MyJourneysDialogProps {
+type MyMembershipsDialogProps = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  data: any[]; // TODO:p Find (create if doesn't exist) type for this.
```
Comment (bobbykolev) [inline src/main/topLevelPages/myDashboard/myMemberships/MyMembershipsDialog.tsx:20]: Please, update this. As far as I follow, it should be one of two types - the old MembershipProps, or the new one where you're passing subspaces.

File: src/main/topLevelPages/myDashboard/DashboardWithMemberships/DashboardSpaces/DashboardSpaces.tsx [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+                />
+              </Card>
+
+              <Gutters disableGap disablePadding style={titleAndDescContainer}>
+                <PageTitle textAlign="center" style={spaceTitle}>
+                  {profile?.displayName}
+                </PageTitle>
+
+                {tagline && (
+                  <Caption
```
[the remark this answers is not in the document]
Comment (reactoholic) [reply src/main/topLevelPages/myDashboard/DashboardWithMemberships/DashboardSpaces/DashboardSpaces.tsx:81]: My bad! I though I was inspecting exatly the SpacePageBanner elements and used the `Caption`, clearly I wasn't inspecting them :)

File: src/main/topLevelPages/myDashboard/DashboardWithMemberships/DashboardSpaces/DashboardSpaces.tsx [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+                })}
+
+                {childMemberships.length > 3 && (
+                  <GridItem columns={cardColumns}>
+                    <Paper
+                      component={Button}
+                      sx={exploreAllButton}
+                      endIcon={<DoubleArrowOutlined />}
+                      onClick={() => {
+                        setSelectedSpaceIdx(idx);
```
[the remark this answers is not in the document]
Comment (reactoholic) [reply src/main/topLevelPages/myDashboard/DashboardWithMemberships/DashboardSpaces/DashboardSpaces.tsx:122]: Agree on that! I'm moving it to the business logic part.

File: src/main/topLevelPages/myDashboard/DashboardWithMemberships/DashboardSpaces/useDashboardSpaces.ts [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+  const [selectedSpaceName, setSelectedSpaceName] = useState('');
+  const [selectedSpaceIdx, setSelectedSpaceIdx] = useState<number | null>(null);
+
+  const { data, loading } = useDashboardWithMembershipsQuery();
+
+  const theme = useTheme();
+
+  const columns = useColumns();
+
+  const { t } = useTranslation();
```
[the remark this answers is not in the document]
Comment (reactoholic) [reply src/main/topLevelPages/myDashboard/DashboardWithMemberships/DashboardSpaces/useDashboardSpaces.ts:22]: You are correct! I forgot to move it there after I removed its use in here. I was using it in both places before and left it here.

**Pull request merged**